src/features/device/pca10153.json

```
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud Multi-Service",
+                "type": "jlink",
+                "description": "Evaluate nRF Cloud Services. Onboarding, data collection, FW updates over the air, logging and alerts.",
```

I think we can bet on users knowing what FOTA is.
```suggestion
                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging, and alerts.",
```